ui/src/app/workflows/components/submit-workflow-panel.tsx

```diff
@@ -115,81 +118,14 @@ export class SubmitWorkflowPanel extends React.Component<Props, State> {
         return null;
     }
 
-    private displaySelectFieldForEnumValues(parameter: Parameter, parameterStateName: ParamSelector) {
-        return (
-            <Select
-                key={parameter.name}
-                value={this.getValue(parameter)}
-                options={parameter.enum.map(value => ({
-                    value,
-                    title: value
-                }))}
-                onChange={event => {
-                    const update = {} as State;
-                    update[parameterStateName] = this.state[parameterStateName].map(p => ({
-                        name: p.name,
-                        value: p.name === parameter.name ? event.value : this.getValue(p),
-                        enum: p.enum
-                    }));
-                    this.setState(update);
-                }}
-            />
-        );
-    }
-
-    private displayInputFieldForSingleValue(parameter: Parameter, parameterStateName: ParamSelector) {
-        return (
-            <textarea
-                className='argo-field'
-                value={this.getValue(parameter)}
-                onChange={event => {
-                    const update = {} as State;
-                    update[parameterStateName] = this.state[parameterStateName].map(p => ({
-                        name: p.name,
-                        value: p.name === parameter.name ? event.target.value : this.getValue(p),
-                        enum: p.enum
-                    }));
-                    this.setState(update);
-                }}
-            />
-        );
-    }
-
-    private renderParameters(parameters: Parameter[], parameterStateName: ParamSelector) {
-        return (
-            <>
-                {parameters.map((parameter, index) => (
-                    <div key={parameter.name + '_' + index} style={{marginBottom: 14}}>
-                        <label>{parameter.name}</label>
-                        {parameter.description && (
-                            <Tooltip content={parameter.description}>
-                                <i className='fa fa-question-circle' style={{marginLeft: 4}} />
-                            </Tooltip>
-                        )}
-                        {(parameter.enum && this.displaySelectFieldForEnumValues(parameter, parameterStateName)) ||
-                            this.displayInputFieldForSingleValue(parameter, parameterStateName)}
-                    </div>
-                ))}
-            </>
-        );
-    }
-
-    private getValue(p: Parameter) {
-        if (p.value === undefined) {
-            return p.default;
-        } else {
-            return p.value;
-        }
-    }
-
     private submit() {
         this.setState({isSubmitting: true});
         services.workflows
             .submit(this.props.kind, this.props.name, this.props.namespace, {
```

Comment:
We can use an endpoint for Archived workflows here, but I think not necessary.

https://github.com/toyamagu-2021/argo-workflows/blob/df85d398af7596f99fcfcfe53688e5edbf1d2f79/server/workflow/workflow_server.go#L634